src/frameworks/drupal7/composer-manager.md

```diff
@@ -36,28 +36,34 @@ Then, manually create the `composer` directory and place a `.gitignore` file ins
 
 ## 3. Update the build hook
 
-If the `composer.json` file is in the application root (as a sibling of the `.platform.app.yaml` file) then it's simplest to just use the `composer` build flavor and let it install Composer dependencies:
+Create a build hook in your `.platform.app.yaml` file that will install Composer dependencies:
 
 ```yaml
-build:
-   flavor: composer
+hooks:
```

Comment:
I can't think of a situation where one would put the `composer.json` at the root of the repository, so I thought it was an unnecessary distraction.